### PR TITLE
prysm web interface 

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

File: compose-examples/prysm-only/docker-compose.yaml [hunk 1]
```diff
@@ -29,6 +29,8 @@ services:
       - geth
     command: --config-file=/config/beacon.yaml
     ports:
+      - 3500:3500/tcp # for web-ui
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:32]: please add `127.0.0.1` to bind on on localhost and make it inacessible from outside

File: compose-examples/prysm-only/docker-compose.yaml [hunk 2]
```diff
@@ -48,6 +50,9 @@ services:
     depends_on:
       - beacon
     command: --config-file=/config/validator.yaml
+    ports:
+      - 8081:8081/tcp # for web-ui
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:54]: please add 127.0.0.1 to bind on on localhost and make it inacessible from outside

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/prysm-only/docker-compose.yaml (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:32]: okey

File: compose-examples/prysm-only/docker-compose.yaml (hunk 2 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:54]: got it

File: compose-examples/prysm-only/docker-compose.yaml [hunk 3]
```diff
@@ -29,7 +29,7 @@ services:
       - geth
     command: --config-file=/config/beacon.yaml
     ports:
-      - 3500:3500/tcp # for web-ui    
+      - 127.0.0.1:3500:3500/tcp # for web-ui    
       - 8080:8080/tcp # for web-ui
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:33]: this too, sorry, with 127.0.0.1

File: compose-examples/prysm-only/docker-compose.yaml [hunk 4]
```diff
@@ -51,7 +51,7 @@ services:
       - beacon
     command: --config-file=/config/validator.yaml
     ports:
-      - 8081:8081/tcp # for web-ui    
+      - 127.0.0.1:8081:8081/tcp # for web-ui    
       - 7500:7500/tcp # for web-ui
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:55]: this too, sorry, with 127.0.0.1

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/prysm-only/docker-compose.yaml (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml]: `127.0.0.1` is added

File: compose-examples/prysm-only/docker-compose.yaml (hunk 2 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml]: `127.0.0.1` is added

File: compose-examples/prysm-only/docker-compose.yaml (hunk 3 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml]: No problem, `127.0.0.1` is added

File: compose-examples/prysm-only/docker-compose.yaml (hunk 4 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml]: `127.0.0.1` is added too

**Review by stefa2k — APPROVED** [review]

**Comment by stefa2k** [comment]
lgtm, thanks

**Pull request merged**